### PR TITLE
Patrick Steveson mvp almost done

### PR DESCRIPTION
**Pull request by Mrsteveson**
Had a bug that took forever to sort out so the styling has not been done. Working on it now.

### CODE REVIEW

**Comment by Mrsteveson** [comment]
Finished up styling so it looks close to the design images.

**Review by NickStrick — APPROVED** [review]
looks good so far, keep up the good work!

File: instagram/src/App.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    });
+
+    this.setState({ filter: filteredPosts })
+  };
+
+  render() {
+    return (
+      <div className="App">
+        <SearchBar
+          search = {this.handleSearch}
```
Comment (NickStrick) [inline instagram/src/App.js]: remember to keep names consistent through out different components

File: instagram/src/components/CommentSection/Comment.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './Comment.css';
+
+const Comment = props => {
+    return (
+        <div className = 'comments'>
+            <p className = 'commentUser'>
+                <button className = 'deleteBtn' id = {props.comment.id} onClick = {props.deleted}>x</button>
```
Comment (NickStrick) [inline instagram/src/components/CommentSection/Comment.js]: wow, even got a delete button, awesome!

File: instagram/src/components/CommentSection/CommentSection.js [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+        // console.log(event.currentTarget.id);
+    }
+
+    render() {
+        return(
+            <div>
+                {this.state.comments && this.state.comments.map(comment => (
+                    <Comment 
+                        comment = {comment}
+                        deleted = {this.deleteComment}
```
Comment (NickStrick) [inline instagram/src/components/CommentSection/CommentSection.js:52]: again be more consistent with your names!

**Review by NickStrick — APPROVED** [review]
Looks great so far! having consistent names for variables throughout your app will make debugging a lot easier and make your code easier to read, other then that keep up the great work.

**Comment by Mrsteveson** [comment]
**note to self**
Refactored all the styling into styled components, some left on their respective js file, while those in for SearchBar are found in SearchBarStyles and imported into SearchBar, both ways work. Might refactor further to change them all to be like that, or might leave it as a reference showing both ways.

**Review by NickStrick — APPROVED** [review]
looks great! got all them components styled and improved upon the login, great job on todays stuff, keep up the great work! your very well prepared for the sprint challenge!

and yeah thats not a bad idea to keep those styles both ways for future reference.

**Review by NickStrick — APPROVED** [review]

**Pull request merged**